AUTHORING_GUIDE.md

```diff
@@ -624,8 +624,32 @@ All the RPCs are inevitably flaky. It can fail for many reasons. The
 `google-cloud` Python client retries requests automatically for most
 cases.
 
-The old api-client doesn't retry automatically, so consider using
-[`backoff`](https://pypi.org/project/backoff/) for retrying. Here is a
+The old api-client doesn't retry automatically, so consider using the Google API Core [`@retry.Retry()`](https://googleapis.dev/python/google-api-core/latest/retry.html) decorator. By default, it will retry [transient errors](https://googleapis.dev/python/google-api-core/latest/retry.html#google.api_core.retry.if_transient_error). Here is an [example](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/4718999884dedcbbf7f3b45c8a9867b644b965da/speech/snippets/quickstart_v2_test.py#L34):
```

Comment:
Please consider submitting `old api-client` with `previous style of client library (api-client)` as described in Google Cloud documentation: https://cloud.google.com/apis/docs/client-libraries-explained#google-api-client-libraries